package/yast2-services-manager.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Mon Aug  6 13:14:05 UTC 2018 - igonzalezsosa@suse.com
+
+- Add support to AutoYaST to set services to be started on demand
+  (related to fate#319428).
+- 4.1.1
```

Comment:
It will conflict with https://github.com/yast/yast-services-manager/pull/167/files#diff-cd1d265cfd0a89451f30ab586c2f78e0R5